NAMESPACE

```diff
@@ -368,6 +384,7 @@ importFrom(data.table,setkeyv)
 importFrom(data.table,setnames)
 importFrom(data.table,setorder)
 importFrom(digest,digest)
+importFrom(downloader,download)
```

Comment:
there should be no reason to use this package in recent versions of R now that the default `download.file` properly handles https urls.